purchase_order_line_description/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+#. Go to **Setting > Users & Companies > Users** and check you have checked
```

Comment:
As I tested, if in the config page, "Use the product purchase description on the purchase order lines" is selected, we can't change "Use only product purchase description on order lines" access right (It always checked).
Should we explain this also in README?